packages/web-react/src/components/Header/README.md

```diff
@@ -1,119 +1,405 @@
-      </ButtonLink>
-    </NavbarActions>
-  </Navbar>
+  <HeaderMobileActions dialogId="header_dialog_example_1" isOpen={isMenuOpen} onOpen={handleMenuOpen} />
+  <HeaderDesktopActions color="primary" aria-label="Main navigation">
+    <HeaderNav>
+      <HeaderNavItem>
+        <HeaderLink href="…" isCurrent>Current menu link</HeaderLink>
+      </HeaderNavItem>
+      <HeaderNavItem>
+        <HeaderLink href="…">Menu link</HeaderLink>
+      </HeaderNavItem>
+      …
+    </HeaderNav>
+  </HeaderDesktopActions>
+  <HeaderDesktopActions color="secondary" aria-label="User area">
+    <HeaderNav>
+      <HeaderNavItem>
+        <HeaderDialogButton
+          onClick={handleUserMenuOpen}
+          aria-controls="header_dialog_example_2"
+          aria-expanded={isUserMenuOpen}
+        >
+          Marian
+        </HeaderDialogButton>
+      </HeaderNavItem>
+    </HeaderNav>
+  </HeaderDesktopActions>
 </Header>
+// HeaderDialog #1
+<HeaderDialog
+  id="header_dialog_example_1"
+  aria-label="Menu"
+  isOpen={isMenuOpen}
+  onClose={handleMenuClose}
+>
+  <HeaderDialogCloseButton />
+  <HeaderDialogActions color="primary" aria-label="Main navigation">
+    <HeaderDialogNav>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…" isCurrent>Current menu link</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…">Menu link</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      …
+    </HeaderDialogNav>
+  </HeaderDialogActions>
+  <HeaderDialogActions color="secondary" aria-label="Menu">
+    <HeaderDialogNav>
+      <HeaderDialogNavItem>
+        <HeaderDialogText UNSAFE_className="text-primary-inverted-disabled">Marian</HeaderDialogText>
+      </HeaderDialogNavItem>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…">Dashboard</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…">Profile</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      …
+      <HeaderDialogNavItem>
+        <HeaderDialogButton>Sign out</HeaderDialogButton>
+      </HeaderDialogNavItem>
+    </HeaderDialogNav>
+  </HeaderDialogActions>
+</HeaderDialog>
+// HeaderDialog #2
+<HeaderDialog
+  id="header_dialog_example_2"
+  aria-label="User menu"
+  isOpen={isUserMenuOpen}
+  onClose={handleUserMenuClose}
+>
+  <HeaderDialogCloseButton />
+  <HeaderDialogActions color="primary" aria-label="User menu">
+    <HeaderDialogNav>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…">Dashboard</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      <HeaderDialogNavItem>
+        <HeaderDialogLink href="…">Profile</HeaderDialogLink>
+      </HeaderDialogNavItem>
+      …
+      <HeaderDialogNavItem>
+        <HeaderDialogButton>Sign out</HeaderDialogButton>
+      </HeaderDialogNavItem>
+    </HeaderDialogNav>
+  </HeaderDialogActions>
+</HeaderDialog>
 ```
 
-### HeaderActions
+## Header (HeaderModern)
```

Comment:
This is old header isn't it? not the modern one

Reply:
This is the new one